lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -310,6 +310,17 @@ def post(self, request):
                     (UserProfile.objects.all().iterator()))
             return self.return_csv('users_{0}.csv'.format(
                 request.META['SERVER_NAME']), header, data)
+
+        elif action == 'download_subscription_emails':
+            emails = PlatformNewsSubscriptionEmail.objects.all().values_list('subscription_email')
+            if emails:
+                return self.return_csv('plarform_updates_subscriptions',
```

Comment:
I guess this method call could fit on a single line. If not, then please, use [these code-style rules](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines).

Reply:
fixed, thx